awx/main/utils/external_logging.py

```diff
@@ -34,7 +35,11 @@ def construct_rsyslog_conf_template(settings=settings):
         'input(type="imuxsock" Socket="' + settings.LOGGING['handlers']['external_logger']['address'] + '" unlink="on")',
         'template(name="awx" type="string" string="%rawmsg-after-pri%")',
     ])
-
+    # Sumologic has a '==' at the end of the host, which should not be parsed by urlparse 
+    # because it we then be parsed again by rsyslogd.  It will be added back below.
+    if log_type == 'sumologic':
```

Comment:
Should we *only* restrict this behavior to sumologic?  What if there's some other HTTP-based endpoint that happens to have an `=` in the path?  It seems like we should circumvent the `=` escaping _regardless_ of the aggregator type (I can't imagine a scenario where we'd *ever* want `=` to turn into `%253D` on the other side of the HTTP request).

Reply:
I agree, we should do this for all logging aggregators, especially now that we are just telling urlparse to not escape `=`'s.